polynote-server/src/main/scala/polynote/server/NotebookSession.scala

```diff
@@ -44,9 +45,9 @@ class NotebookSession(
       env       <- ZIO.environment[SessionEnv]
       _         <- sendNotebookInfo()
       processor <- process(input, output)
-      fiber     <- processor.interruptWhen(closed.await.either).compile.drain.ignore.fork
+      fiber     <- processor.interruptWhen(closed.await.either.as(Either.right[Throwable, Unit](()))).compile.drain.ignore.fork
       keepalive <- Stream.awakeEvery[Task](Duration(10, SECONDS)).map(_ => WebSocketFrame.Ping())
-        .interruptWhen(closed.await.either)
+        .interruptWhen(closed.await.either.as(Either.right[Throwable, Unit](())))
```

Comment:
So is this essentially the same as `closed.await.ignore.map(Either.right[Throwable, Unit])`? (not suggesting a replacement, just checking my understanding)